### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by paulSambolin**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,8 +1,18 @@
+sudo: required
```
Comment (ironmanwaring) [inline .travis.yml]: we only need sudo for the angular-cli - no need for it here, it slows the env provisioning down

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,18 @@
...
 language: node_js
 node_js:
-    - 6
+  - 4.3.2
```
Comment (ironmanwaring) [inline .travis.yml]: also fine to have - node - it's the runtime the travis container uses, has no bearing on what's being deployed to aws

File: .travis.yml [hunk 3]
```diff
@@ -1,8 +1,18 @@
...
+cache:
+  - directories:
+      - node_modules
+      - $(npm config get prefix)/bin/serverless
     
 script:
-    - npm run build
+  # I'm sure there's a better way to do the line below, but I don't know what it is and don't care to figure it out
+  - export STAGE=$(if [ $TRAVIS_BRANCH == "master" ]; then echo "prod"; else AUTHOR_NAME=$(git log -1 $TRAVIS_COMMIT --pretty="%aN") && CAPITAL_FIRST_NAME=${AUTHOR_NAME%% *} && FIRST_NAME=${CAPITAL_FIRST_NAME,,} && echo $FIRST_NAME; fi)
+  - serverless deploy
```
Comment (ironmanwaring) [inline .travis.yml]: is there a reason we aren't deploying to a particular stage?

File: .travis.yml (hunk 2 above)
Comment (paulSambolin) [inline .travis.yml]: Still node, just the version aws lambda uses 4.3.2

File: .travis.yml (hunk 3 above)
Comment (paulSambolin) [inline .travis.yml]: we are deploying to $STAGE stage, no need to declare it on the cli

**Pull request merged**